src/components/Identity.tsx

```diff
@@ -1,63 +1,86 @@
 import React from "react";
-import { KeyPair } from "many";
-import { getFormValue, handleForm } from "../utils";
-
-import Button from "./Button";
-import ButtonGroup from "./ButtonGroup";
-import Section from "./Section";
-import Tabs from "./Tabs";
-import Tab from "./Tab";
+import { Ed25519KeyPairIdentity as Id } from "@liftedinit/many-js";
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  Heading,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Textarea,
+} from "@liftedinit/ui";
 
 interface IdentityProps {
-  setKeys: (keys?: KeyPair) => void;
+  setId: (id?: Id) => void;
 }
 
-const importMnemonic = async (form: HTMLFormElement) => {
-  const mnemonic = getFormValue(form, "mnemonic");
-  return KeyPair.fromMnemonic(mnemonic);
-};
-
-const importPem = async (form: HTMLFormElement) => {
-  const pem = getFormValue(form, "pem");
-  return KeyPair.fromPem(pem);
-};
-
-function Identity({ setKeys }: IdentityProps) {
-  const [activeTab, setActiveTab] = React.useState(0);
-  const [mnemonic, setMnemonic] = React.useState(KeyPair.getMnemonic());
+function Identity({ setId }: IdentityProps) {
+  const [mnemonic, setMnemonic] = React.useState(Id.getMnemonic());
+  const [textarea, setTextarea] = React.useState("");
   return (
-    <Section title="Identity">
-      <ButtonGroup tab={activeTab} setTab={setActiveTab}>
-        <Button label="Anonymous" onClick={() => setKeys()} />
-        <Button
-          label="Random"
-          onClick={() => setMnemonic(KeyPair.getMnemonic())}
-        />
-        <Button label="Seed Words" />
-        <Button label="Import PEM" />
-      </ButtonGroup>
-      <Tabs tab={activeTab}>
-        <Tab />
-        <Tab>
-          <form onSubmit={handleForm(importMnemonic, setKeys)}>
-            <textarea name="mnemonic" disabled value={mnemonic} />
-            <button>Import</button>
-          </form>
-        </Tab>
-        <Tab>
-          <form onSubmit={handleForm(importMnemonic, setKeys)}>
-            <textarea name="mnemonic" />
-            <button>Import</button>
-          </form>
-        </Tab>
-        <Tab>
-          <form onSubmit={handleForm(importPem, setKeys)}>
-            <textarea name="pem" />
-            <button>Import</button>
-          </form>
-        </Tab>
+    <Box bg="white" p={6}>
+      <Heading>Identity</Heading>
+      <Tabs>
+        <TabList>
+          <Tab onClick={() => setId()}>Anonymous</Tab>
+          <Tab onClick={() => setMnemonic(Id.getMnemonic())}>Random</Tab>
+          <Tab>Seed Words</Tab>
+          <Tab>Import PEM</Tab>
```

Comment:
Currently you can generate an identity from these four sources. Once the "Account Selector" dropdown is properly abstracted from Alberto, we can use that instead. Right now it's connected to all sorts of app state.